.env

```diff
@@ -0,0 +1,11 @@
+# Please note that this is an example .env file, and you should
+# edit this file to use your own values.
+#
+# Do not commit your values in this file to the repo, but do
+# commit new keys to this example file
+
+FLASK_APP=app
+FLASK_RUN_PORT=5000
+SERVER_NAME=0.0.0.0
+DATABASE_URL='sqlite:///database.db'
+SECRET_KEY='e9cac0f3f4Yd47a3be91d7b8f5'
```

Comment:
1. You can consider splitting this .env file into 2 files: a .env file and a .flaskenv file for more clarity. It's totally a stylistic choice though.
2.  Given that you are defining environment variables here, you can also put any other env variables needed to run the flask app to these env files so that we can start the server with fewer commands (like only with `python3 -m flask run`)